build/yamls/base/conf/antrea-controller.conf

```diff
@@ -53,3 +53,22 @@ featureGates:
 # API groups. After adding the annotation, legacy CRDs can be deleted safely without impacting
 # new CRDs.
 #legacyCRDMirroring: true
+
+# Enable the integrated node IPAM controller within the Antrea controller.
+#enableNodeIPAMController: false
+
+# Cluster IP CIDR ranges for Pods. Required when enableNodeIPAMController is enabled, and should be set with
+# the value value specified by --cluster-cidr for kube-apiserver.
+#clusterCIDRs: 172.18.0.0/16
```

Comment:
The commented value is the default value for other options. I think 172.18.0.0/16 is an example, not default value. Maybe leave it empty and move the example to above comment to follow the convention?

Reply:
Yep I'll remove along with the defaults.

Reply:
@tnqn @antoninbas BTW should the mask sized be defaulted?

Reply:
I think it should be defaulted to same value as kube-controller-manager

Reply:
I would not default anything, for the same reason you reported below about not defaulting CIDRs.
a /24 default slice on a /25 CIDR isn't a wise solution too...

Reply:
Makes sense, I'll remove the defaults entirely